DESCRIPTION

```diff
@@ -184,3 +184,200 @@ Language: en-US
 RoxygenNote: 7.1.2
 Config/testthat/edition: 3
 Roxygen: list(markdown = TRUE)
+Collate: 
```

Comment:
Haven't been through all the changes, but still curious as to why we need to start using the `Collage` field now?

Does the order in which files are loaded matter now?

I thought this field is [_almost_ never](https://stackoverflow.com/questions/10526392/collate-field-in-package-description) needed.

Reply:
I will probably won't need `Collate` anymore when comes merging time. As I noted above, this is still a WIP, and not ready for review yet. The only reason I opened a PR was to get the tests.